src/api/messages.js

```diff
@@ -0,0 +1,57 @@
+import request from './request'
+import { struct } from 'superstruct'
+
+const Message = struct({
+  text: 'string',
+  sendAt: 'date',
+  userId: 'number',
+  User: 'Object',
```

Comment:
Is the user really necessary here? Maybe we should only send the name of the sender?